asapdiscovery-docking/asapdiscovery/docking/scorer_v2.py

```diff
@@ -21,7 +21,7 @@
 )
 from asapdiscovery.docking.docking_v2 import DockingResult
 from asapdiscovery.ml.inference import InferenceBase, get_inference_cls_from_model_type
-from asapdiscovery.ml.models import MLModelType
```

Comment:
Can we have MLModelType continue to exist, as we may want models that are not `mtenn` based, but we can just subclass the Mtenn one for now. 

eg 

```
class MLModelType(mtenn.config.ModelType)
# no args
```

Reply:
looks like making a subclass of a subclass of an Enum is not allowed: https://docs.python.org/3/howto/enum.html#restricted-enum-subclassing. we can just shadow the `mtenn` Enum here (eg `import mtenn.config.ModelType as MLModelType`), but not really sure if that would accomplish anything